jobqueue/scheduler/kubernetes.go

```diff
@@ -162,22 +161,17 @@ func (c *ConfigKubernetes) GetServerKeepTime() time.Duration {
 
 // Set up prerequisites, call Run() Create channels to pass requests to the
 // controller. Create queue.
-func (s *k8s) initialize(config interface{}, logger log15.Logger) error {
+func (s *k8s) initialize(ctx context.Context, config interface{}) error {
 	s.config = config.(*ConfigKubernetes)
 
-	s.Logger = logger.New("scheduler", "kubernetes")
 	kubeLogFile := filepath.Join(s.config.ManagerDir, kubeSchedulerLog)
-	fh, err := log15.FileHandler(kubeLogFile, log15.LogfmtFormat())
-	if err != nil {
-		return fmt.Errorf("wr kubernetes scheduler could not log to %s: %s", kubeLogFile, err)
-	}
 
-	l15h.AddHandler(s.Logger, fh)
+	clog.ToFileAtLevel(kubeLogFile, "warn")
```

Comment:
This is something we can't do, and is a problem with having a single global logger. We need the ability to have some logs going to one place, and other logs going to another place.
As a simple fix, just continue to use log15 directly here, so largely undo the change.